Source/Core/AudioCommon/CubebUtils.h

```diff
@@ -6,8 +6,26 @@
 
 #include <memory>
 
+#ifdef _WIN32
+#include <comdef.h>
+#endif
+
 struct cubeb;
 
+// Helper to initialize and uninitialize thread access for the libraries that cubeb uses
+struct ScopedThreadAccess
```

Comment:
```suggestion
class ScopedThreadAccess
```

This should also be within the CubebUtils namespace below.

Reply:
I moved it. I don't see how class or struct is relevant. I "copied" it from cubeb itself and it was a struct there.

Reply:
See https://github.com/dolphin-emu/dolphin/blob/master/Contributing.md#classes-and-structs for `class` vs `struct`. I agree with Lioncash here (not that it actually makes a huge difference which one you use).